lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,73 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
+        def initialize(serializer, options = {})
+          options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
+            .include_args_to_hash(options[:include])
+
+          super(serializer, options)
```

Comment:
Or https://github.com/rails-api/active_model_serializers/pull/1050/files#diff-ae1e38e96a8bf28ae37918a65c448c73R96 where I called `included_for` just when I used it